client/src/app/pages/applications/components/application-detail-drawer/application-risk.tsx

```diff
@@ -36,5 +36,5 @@ export const ApplicationRisk: React.FC<IApplicationRiskProps> = ({
     );
   }
 
-  return <RiskLabel risk={assessments[0].risk || "unknown"} />;
+  return <RiskLabel risk={application.risk || "unknown"} />;
```

Comment:
With `risk` arriving on the application from hub, and only being shown from there, is any of the assessment fetching still needed?  Seems like the `ApplicationRisk` component may not be needed any longer.